pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[tool.black]
+target-version = ['py311']
+
+# This pyproject.toml is setup so it can be used with or without poetry and also
+# supports editable installs (PEP 660) without breaking IDE and linter inspection.
+
+# DEVELOPER INSTALL INSTRUCTIONS (assuming current directory is at the top of the project) :
+# Poetry:
+#   poetry install --extras dev
+# PIP:
+#   pip install -e .[dev]
+
+[tool.poetry]
+name = "screenpy_pyotp"
+version = "4.0.1"
+description = "ScreenPy extension to enable interacting with PyOTP."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+maintainers = ["Marcel Wilson"]
+license = "MIT"
+repository = "https://github.com/ScreenPyHQ/screenpy_pyotp"
+documentation = "https://screenpy-pyotp-docs.readthedocs.io"
+readme = "README.md"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: BDD",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+]
+
+# It's possible to add optional dependencies with the poetry CLI tool using:
+#   poetry add --optional some_dev_pkg
+# This will set `optional = true` flag in [tool.poetry.dependencies], as seen below
+# But then you need to remember to manually edit the [tool.poetry.extras] dev section
+# and declare the package. This allows `pip install .[dev]` to work as expected
+# Poetry users will need to use the `--extras dev` option rather than the `--with dev`
+# so we dont have two different sets of package versions to update.
+
+[tool.poetry.dependencies]
+python = "^3.8"
+screenpy = ">=4.0.2"
+pyotp = ">=2.6.0"
+typing-extensions = ">=4.1.1"
```

Comment:
Do we actually use `typing-extensions` in this package? I think we can remove this line.

Reply:
Looking through it... i kinda think we could also remove the `screenpy` dependency, too, hilariously.

Reply:
It is pretty weird... but it's not a dependency, so i think we should remove it to keep it clean.